README.md

```diff
@@ -119,6 +119,7 @@ Global development tools:
 - [Prettier](./.prettierrc) to use the same code style formatting.
 - [TypeScript](./tsconfig.json) for strict type checking.
 - [ESLint](./eslint.config.js) to check for popular mistakes in JavaScript.
+- [remark](./.remarkrc.js) to find mistakes in .md files.
```

Comment:
```suggestion
- [remark](./.remarkrc.js) to find mistakes in `.md` files.
```

1. Better formatting is better for readability.
2. File names should be in `<code>` according to practice in the web.
3. Without `<code>` the filename dot conflicts with typography dot.